tests/config/pytest_modules.yml

```diff
@@ -1120,10 +1120,6 @@ fgbio/callmolecularconsensusreads:
   - modules/nf-core/fgbio/callmolecularconsensusreads/**
   - tests/modules/nf-core/fgbio/callmolecularconsensusreads/**
 
-fgbio/fastqtobam:
-  - modules/nf-core/fgbio/fastqtobam/**
-  - tests/modules/nf-core/fgbio/fastqtobam/**
-
```

Comment:
please remove also the old pytest files themselves.

Reply:
Done!